src/Sylius/Component/Core/Dashboard/DashboardStatistics.php

```diff
@@ -15,6 +15,13 @@
 
 use Sylius\Component\Core\Model\ChannelInterface;
 
+trigger_deprecation(
+    'sylius/core',
+    '1.14',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0.',
+    DashboardStatistics::class,
```

Comment:
```suggestion
    self::class,
```
and in other classes, wdyt?

Reply:
It want work: Internal error: Cannot use "self" in the global scope